src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -31,7 +31,7 @@ import scala.concurrent.duration._
 import scala.concurrent.{ExecutionContext, Future}
 
 case class ClusterNotReadyException(googleProject: GoogleProject, clusterName: ClusterName)
-  extends LeoException(s"Cluster ${googleProject.value}/${clusterName.value} is not ready yet, chill out and try again later", StatusCodes.EnhanceYourCalm)
+  extends LeoException(s"Cluster ${googleProject.value}/${clusterName.value} is not ready yet. It may be updating, try again later", StatusCodes.UnprocessableEntity)
```

Comment:
I don't think cluster not being ready is a `UnprocessableEntity`. My understanding of `UnprocessableEntity` is your request body is mal formatted

Reply:
If we're changing error code, we should also check if we need to update swagger file

Reply:
Good point, +1